### PR TITLE
Creating form

### PR DESCRIPTION
**Pull request by iKrystyan**
In this milestone, I added
1, HTML for the form
2. CSS

### CODE REVIEW

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Christianib003, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] The student wrote at least 2 comments in the review.
- [ ] The student used the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments.

- Make sure the student left the comment under a line or block of code that fits the context the comment is about.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          <input type="email"name="email" id="email"class="form-control"placeholder="Enter your Email"required/>
+          <br>
+          
+          <label id="number-label" for="number">Age<span>(optional)</span></label>
+          <br>
+          <input type="number" name="age" id="number" min="10" max="99" class="form-control" placeholder="Age" />
+         
+          <br>
+          <p>Which option best describes your current role?</p>
+          <select id="dropdown" name="role" class="form-control" required>
```
Comment (iKrystyan) [inline index.html:34]: Nice work adding the dropdown

File: index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            <option disabled selected value>Select current role</option>
+            <option value="student">Student</option>
+            <option value="Full Time Job">Full Time Job</option>
+            <option value="Full Time Learner">Full Time Learner</option>
+            <option value="Prefer not to say" >Prefer not to say</option>
+            <option value="other">Other</option>
+          </select>
+            <p>Would you recommend freeCodeCamp to a friend?</p>
+          
+            <input name="recommend" value="definitely" type="radio" class="input-radio" checked/>Definitely
```
Comment (iKrystyan) [inline index.html:44]: Well done in implementing radio input.

**Review by iKrystyan — COMMENTED** [review]
You have done a good job in Implementing the input types. Good Luck and Happy Coding!!!

**Review by Lordkaito — APPROVED** [review]
Hi @Christianib003,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] [OPTIONAL] When making a review for any partner, it is good and common to explain why you requesting changes or why you think something is good or bad. For instance, instead of saying "Change this" you can say "I think it would be good to refactor this because x". What I'm trying to say, is that you should start building good communication skills in order to improve communication with your partner or any other developer and avoid conflicts.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**